pkg/apis/application/v1alpha1/types.go

```diff
@@ -1212,6 +1212,15 @@ type RevisionMetadata struct {
 	SignatureInfo string `json:"signatureInfo,omitempty" protobuf:"bytes,5,opt,name=signatureInfo"`
 }
 
+// ChartDetails contains helm chart metadata for a specific version
+type ChartDetails struct {
+	Description string `json:"description,omitempty" protobuf:"bytes,1,opt,name=description"`
+	// Home string
+	Home string `json:"home,omitempty" protobuf:"bytes,2,opt,name=home"`
+	// Concatenated maintainers, name and email, e.g. "John Doe <john_doe@my-company.com>"
+	Maintainers string `json:"maintainers,omitempty" protobuf:"bytes,3,opt,name=maintainers"`
```

Comment:
The official helm chart documents it as a list of objects:
```
maintainers: # (optional)
  - name: The maintainers name (required for each maintainer)
    email: The maintainers email (optional for each maintainer)
    url: A URL for the maintainer (optional for each maintainer)
``` 
I can add that instead of the concatenated list of strings, in order to be faithful to the original.

The intent here was to keep it simple, as I can't see a need for such detailed information. Nevertheless, maybe the need will come later on.